runserver.py

```diff
@@ -46,5 +50,5 @@
 
 
 from anitya.app import APP
-APP.debug = True
```

Comment:
I'm actually :-1: to this, runserver is meant for dev not for actually running the app, so debugging here is pretty useful :)